tests/testthat/_snaps/json.md

```diff
@@ -0,0 +1,88 @@
+# json joins work
+
+    Code
+      packed
+    Output
+      # A tibble: 3 x 3
+         col1 key   df2       
+        <int> <chr> <list>    
+      1     1 a     <json [1]>
```

Comment:
Should we return a plain character here instead, so that a preview is shown when printed?

Reply:
done in https://github.com/cynkra/dm/pull/918/commits/35d870958c1cc6f9fd00ca0d97c8f8406e63b944